hi-ml-multimodal/notebooks/phrase_grounding.ipynb

```diff
@@ -12,6 +12,32 @@
     "#  -------------------------------------------------------------------------------------------"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Phrase grounding\n",
+    "\n",
+    "This notebook demonstrates the multimodal models introduced in our ECCV 2022 paper:\n",
+    "\n",
+    "> Boecking, B., Usuyama, N., Bannur, S., Castro, D., Schwaighofer, A., Hyland, S., Wetscherek, M., Naumann, T., Nori, A., Alvarez-Valle, J., Poon, H., & Oktay, O. (2022). *Making the Most of Text Semantics to Improve Biomedical Vision–Language Processing* ([preprint](https://arxiv.org/abs/2204.09817))\n",
```

Comment:
It might be good to emphasise that the notebook is solely for research evaluation purposes, not intended to be medical product or clinical use.